src/test/components/Button.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import render from 'react-test-renderer';
```

Comment:
- [ ] Please avoid using react-test-renderer in this project and instead use the one mentioned in the requirements which is: @testing-library/react to learn a new testing tool :)